src/commcare_cloud/ansible/roles/cloudwatch_logs/templates/cloudwatch_logs.json.j2

```diff
@@ -0,0 +1,19 @@
+{
+    "logs":
+    {
+        "logs_collected": {
+            "files": {
+                "collect_list": [
+                {% for cfg in log_files_collect_list -%}
+                {
+                    "file_path": "{{ cfg.file_path }}",
+                    "log_group_name": "{{ cfg.log_group_name }}",
+                    "log_stream_name": "{instance_id}-{local_hostname}-{{ cfg.log_stream_name_suffix }}"
```

Comment:
Does this mean that groups that streams that don't use a suffix will just end in '-'? That's a little odd if you ask me. Perhaps we can use a suffix like `"combined_logs"` for log streams that include a bunch of different log file patterns (*.log*)

Reply:
Yeah I hoped no one would notice/it wouldn't be a big deal. I'll look into jinja syntax to see if I can remove the `-` if there's no `log_stram_nuame_suffix`